images/nodesvc-base/Dockerfile

```diff
@@ -1,20 +1,43 @@
-FROM ghcr.io/scality/federation/nodesvc-base:7.10.5.0
-
-ENV UTAPI_CONFIG_FILE=${CONF_DIR}/config.json
+ARG DEBIAN_FRONTEND=noninteractive
+ARG UTAPI_VERSION=latest
+FROM ghcr.io/scality/utapi:${UTAPI_VERSION} AS base
```

Comment:
So we really need 2 images for utapi ? Can we use only 1 image ? The other one doesn't use the same node version

Reply:
I think we need two, because UTAPI is used on both ARTESCA and RING:
- Artesca uses the default image and does not use `supervisord` to starts its container
- I don't think they use `ballot` on artesca side
- They also don't have this requirement of having CONF_DIR DATA_DIR and all that stuff.

I can match the node version and image though.

Reply:
Also I might have misunderstood what you meant, did you meant to use only the `FROM ghcr.io/scality/utapi:${UTAPI_VERSION}` and install and configure the necessary thing on top of it instead of using a multi stage build?

Reply:
No, multi stage is better we've got things we don't need in the other image.

But can we keep the same nodejs version in every repo for the final stage.

Maybe if we can't touch the version for the artesca one we can have a local build stage if it's only to `yarn install` we don't need to be dependant on the other image

Reply:
I have aligned the two images with the same base image.